src/hotspot/share/memory/virtualspace.cpp

```diff
@@ -169,14 +169,14 @@ static char* reserve_memory(char* requested_address, const size_t size,
 }
 
 static char* reserve_memory_special(char* requested_address, const size_t size,
-                                    const size_t alignment,  bool exec) {
+                                    const size_t alignment, const size_t page_size, bool exec) {
```

Comment:
I don't think the const makes sense... nor for the other arguments (but when in Rome...)

Reply:
Yes, I'm staying in Rome :) But we could certainly clean it up later.